obfuscation/ufc.dto.ts

```diff
@@ -0,0 +1,98 @@
+export interface ITargetingRuleCondition {
```

Comment:
Would it be useful to get these types from the Eppo js-common-sdk package?

Reply:
It would be nice. But I couldn't find them in https://github.com/Eppo-exp/js-sdk-common.